### ISSUE TITLE
Request to add an option starting from xx

### ISSUE THREAD

**Issue by wutaijieing** [PKQYPKJ/HUAWEI-HONOR-bootload-unlocker#1]
Request to add an option starting from xx and record the attempted unlock code every once in a while (overlay)

**Comment by PKQandBaaad** [comment]
Of course, if I have time, I will add a fork to this account and add this content, and by the way, add a new version of the unlock code rules, that is, with letters (PKQYPKJ cannot log in due to 2FA)

**Comment by PKQandBaaad** [comment]
Considering people from other countries, I can also add multilingual translations.
